### DISCUSSION TITLE
Why The MassTransitInstrument does not log Transport shutdown.

### DISCUSSION THREAD

**Discussion by persiandeveloper** [open-telemetry/community#793] [Community Discussions]
In asp.net core 5.0 app with OpenTelemetry.Contrib.Instrumentation.MasssTrasnit 1.0.0.-beta-1, when the a transport shuts down, or goes offline, the OpenTelemetry does not log anything related t that. 

Should I enable any specific configuration for that to work ?

**Comment by SergeyKanzhelev** [comment]
CC:  @open-telemetry/dotnet-contrib-approvers @alexvaluyskiy

(from: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/cbd54228112a55899cda18f3f47ec6782b43febb/CODEOWNERS#L11)

@persiandeveloper can you please clarify, you do not see "failed" client Spans when calling into the library which is in "broken" state? Or you want a separate notification of some form?

**Reply by SergeyKanzhelev** [reply]
Also you can post this as an issue on the corresponding repository. Be sure to include expected and actual behavior and ideally a code example or step by step repro

**Reply by persiandeveloper** [reply]
Hi @SergeyKanzhelev 

This is the code I use to configure the OpenTelemetry : 

```
services.AddOpenTelemetryTracing(
               (builder) => builder
                    .AddSource("webapp")
                    .SetResourceBuilder(
                      ResourceBuilder.CreateDefault().AddService(
                          serviceName: "test-web-app", serviceVersion: "1.0"))
                    .AddAspNetCoreInstrumentation()
                    .AddHttpClientInstrumentation()
                    .AddMassTransitInstrumentation()
                    .SetErrorStatusOnException()
                    .AddConsoleExporter()
                   );
```

And here it is action of a controller code that I call the Send from MassTransit : 

```
var endPoint = await _bus.GetEndpoint(....);

await endPoint.Send(message, cancellationToken);
```

if the Transport fails just before calling the "Send", it will not be tracked by OpenTelemetry.
